### ISSUE TITLE
Error evaluating generator expression: $<TARGET_FILE:modal_progress_hud_nsn_plugin>

### ISSUE THREAD

**Issue by mefaiz** [kphanipavan/modal_progress_hud_nsn#6]
Hi, i want to run my flutter project on desktop. But i have one issue when run it. Any guess how to solve it?

![cmake error](https://user-images.githubusercontent.com/67181856/198454956-028900e6-127c-4247-ba07-e24c8c014d01.jpg)

**Comment by kphanipavan** [comment]
Hello...

Have you tried running the example app/fresh app with just this plugin? See if the problem persists in it...

If it does, please try alternatives:
https://pub.dev/packages/modal_progress_hud_alt
https://pub.dev/packages/blurry_modal_progress_hud

I don't have a windows machine at the moment to debug this issue. Will try to replicate it on my end push the fix accordingly as soon as possible.

Edit:
Try `flutter clean` first

**Comment by mefaiz** [comment]
Hi.. i have tried running on fresh app and the error still exist.

Edit:
right now i tried https://pub.dev/packages/modal_progress_hud_alt for alternative for my current project. works well

**Comment by adromil** [comment]
Hi @kphanipavan also encountered the same issue while trying to run my project as a Windows Desktop.

**Comment by kphanipavan** [comment]
Will push a fix soon, try using the git version after the push...
Thank you for the info.

**Comment by kphanipavan** [comment]
@adromil 
@acheyfaiz 
Try below:

Replace
```dart
dependencies:
...
  modal_progress_hud_nsn: ^0.3.0
...
```
 with 
```dart
dependencies:
...
  modal_progress_hud_nsn:
    git:
      url: https://github.com/kphanipavan/modal_progress_hud_nsn.git
      ref: master
...
```

Do comment on the result.

**Comment by kphanipavan** [comment]
Hii,
@acheyfaiz @adromil 

Included this fix to stable version [0.4.0](https://pub.dev/packages/modal_progress_hud_nsn). Closing this for now, reopen if necessary.

Thank you.
~ Pvn

**Comment by Ut321** [comment]
Flutter clean
Flutter pub get 
Flutter run 

_________________
Flutter upgrade
Flutter clean
Flutter pub upgrade
Flutter run 

I think then it might will solve the issue .
